editor/src/components/canvas/canvas-strategies/keyboard-absolute-move-strategy.ts

```diff
@@ -0,0 +1,101 @@
+import { MetadataUtils } from '../../../core/model/element-metadata-utils'
+import { stylePropPathMappingFn } from '../../inspector/common/property-path-hooks'
+import { adjustNumberProperty } from '../commands/adjust-number-command'
+import { wildcardPatch } from '../commands/wildcard-patch-command'
+import { Keyboard } from '../../../utils/keyboard'
+import { CanvasStrategy } from './canvas-strategy-types'
+
+export const keyboardAbsoluteMoveStrategy: CanvasStrategy = {
+  id: 'KEYBOARD_ABSOLUTE_MOVE',
+  name: 'Keyboard absolute Move',
+  isApplicable: (canvasState, _interactionState, metadata) => {
```

Comment:
Are `isApplicable` and `fitness` the same as the mouse based ones for the current absolute move strategy, with the exception of checking the key that is pressed?

Reply:
yes, they are basically the same